crypto/rsa/rsa_chk.c

```diff
@@ -20,7 +20,8 @@ int RSA_check_key_ex(const RSA *key, BN_GENCB *cb)
 {
     BIGNUM *i, *j, *k, *l, *m;
     BN_CTX *ctx;
-    int ret = 1;
+    int ret = 1, primes = 0, idx;
```

Comment:
Come on, man. Is it too much to ask to check other occurrences? I mean view a comment in wider context than one line. `ex_primes`... And question is and should have been are there other occurrences left?

Reply:
All addressed...